libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -815,14 +979,26 @@ class MapboxRouteLineApi(
     fun showRouteWithLegIndexHighlighted(
         legIndexToHighlight: Int,
         consumer: MapboxNavigationConsumer<Expected<RouteLineError, RouteLineUpdateValue>>
+    ) {
+        showRouteWithLegIndexHighlighted(legIndexToHighlight, null, consumer)
+    }
```

Comment:
Shouldn't usage of this function reset `activeLegIndex`? What are other side effects of using this function while in active guidance with the current setup?

Reply:
This method doesn't change any state, not in the current release and not with the modifications in this PR.  Its purpose is to get a view of the route line with a specific leg active.  It's similar to `getRotueDrawData` it just gives a view of the route line in its current state. The docs explicitly indicate this method makes no state changes.